tutorials/for-developers-abstractmcmc-turing/Manifest.toml

```diff
@@ -1,1591 +1,1652 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.6.7"
```

Comment:
@willtebbutt This Manifest file is wrong, isn't it? Currently only Julia 1.6 is supported (see eg #394).

Reply:
It seems it was generated in https://buildkite.com/julialang/turingtutorials/builds/675#01892bfc-531f-4454-ba37-1a9e3885e679 - with Julia 1.6. Apparently 1.6 is not picky about instantiating environments with Manifest files not generated explicitly for 1.6 but 1.6 will generate (and probably use) a different Manifest file. So it didn't break anything (it seems) but to me it seems it's not the "correct" Manifest file (as generally they are not guaranteed to work with different Julia versions).